pkg/cmd/cli/secrets/source.go

```diff
@@ -0,0 +1,205 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	CreateSourceSecretRecommendedCommandName = "new-sourcecfg"
+
+	createSourceSecretLong = `
+Create a new source secret
+
+Source secrets are used to authenticate against SCM server.
+
+When creating applications, you may have a SCM server that requires authentication. In order for the
+nodes to clone source code on your behalf, they have to have the credentials. you can provide this
+information, by creating a sourcecfg secret and attaching it to your service account.
```

Comment:
"**Y**ou can provide this **information by** creating..." (capital Y and no comma)